client/src/pages/dashboard/Dashboard.js

```diff
@@ -89,7 +94,13 @@ class Dashboard extends Component {
     event.preventDefault();
     API.handleUpVote(projectId)
       .then(() => {
-        this.getAllProjects();
+        API.recordVotedProject(this.state.userId, projectId)
```

Comment:
Once the handle vote request succeeds, we can record the projectId to the user using the userId state. This lets us know who triggered it.